courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2565 @@
+{
```

Comment:
<p>formatting. line too long</p><p>is it possible to not hard code the hash_bucket_size?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
<p>Re line too long:  Agreed - I don't see a away to break up the nested function.</p><p> </p><p>Re:  hard coding the hash_bucket size.  After a bit of research, I do not see an "elegant" way of doing this.  Here is one example:  </p><pre class="ql-syntax" spellcheck="false">thal_hashed = feature_column.categorical_column_with_hash_bucket(
&nbsp; &nbsp; &nbsp; 'thal', hash_bucket_size=1000)
demo(feature_column.indicator_column(thal_hashed))
</pre><p> </p>